docs/Developer-Guide.md

```diff
@@ -85,6 +85,46 @@ You need to install the following to build Kata Containers components:
 - `make`.
 - `gcc` (required for building the shim and runtime).
 
+# Install a hypervisor
+
```

Comment:
Could you split this commit in two, one with the code move, the other with the actual text changes? As is, it's dififcult to review what you actually changed.

Reply:
Done. Made the rationale per commit a little clearer.